src/elements/content-sidebar/activity-feed/activity-feed/__tests__/ActivityThread.test.js

```diff
@@ -1,22 +1,74 @@
 // @flow
 import React from 'react';
-import { shallow, ShallowWrapper } from 'enzyme';
-
+import { render, screen, fireEvent } from '@testing-library/react';
+import { IntlProvider } from 'react-intl';
 import ActivityThread from '../ActivityThread.js';
+import replies from '../../../../../__mocks__/replies';
 
+jest.mock('react-intl', () => jest.requireActual('react-intl'));
 describe('src/elements/content-sidebar/activity-feed/activity-feed/ActivityThread', () => {
-    const getWrapper = (props = {}): ShallowWrapper => shallow(<ActivityThread {...props}>Test</ActivityThread>);
+    const defaultProps = {
+        replies,
+        totalReplyCount: 2,
+        hasReplies: true,
+    };
+
+    function customRender(ui, { locale = 'en', ...renderOptions } = {}) {
+        function Wrapper({ children }: { children: React.Component }) {
+            return <IntlProvider locale={locale}>{children}</IntlProvider>;
+        }
+        return render(ui, { wrapper: Wrapper, ...renderOptions });
+    }
+
+    const getWrapper = props =>
+        customRender(
+            <ActivityThread {...defaultProps} {...props}>
+                Test
+            </ActivityThread>,
+        );
+
+    test('should render children component wrapped in ActivityThread if hasReplies is true', () => {
+        getWrapper();
+
+        expect(screen.queryByTestId('activity-thread')).toBeInTheDocument();
+        expect(screen.getByText('Test')).toBeVisible();
+    });
+
+    test('should render children component if hasReplies is false', () => {
+        const { getByText } = getWrapper({ hasReplies: false });
+
+        expect(screen.queryByTestId('activity-thread')).not.toBeInTheDocument();
+        expect(getByText('Test')).toBeVisible();
+    });
+
+    test('should call onGetReplies on button click', () => {
+        const onGetReplies = jest.fn();
+        const { getByText } = getWrapper({ onGetReplies });
+
+        const button = getByText('See 1 reply');
```

Comment:
Do we have a `localize` helper similar to ones we see in other internal codebases?

Reply:
I found it in /`test/support/i18n.js `- thanks!